roles/openshift-emptydir-quota/tasks/main.yaml

```diff
@@ -3,7 +3,7 @@
   replace:
     dest: /etc/origin/node/node-config.yaml
     regexp: '^(.*)perFSGroup: (\s+.*)?$'
-    replace: '\1 perFSGroup: 512Mi\2'
+    replace: '\1 perFSGroup: {{ openshift_node_local_quota_per_fsgroup }}\2'
```

Comment:
This is done during the openshift install on aws. I believe we should deprecate role for gce/vmw and let the installer handle it.